play.pokemonshowdown.com/src/panel-topbar.tsx

```diff
@@ -150,10 +150,33 @@ export class PSHeader extends preact.Component<{ style: object }> {
 			{closeButton}
 		</li>;
 	}
+	handleRoomTabOverflow() {
+		if (!PS.rightPanel) return; // TODO handle vertical overflow
+		const avaliableSpace = PS.rightPanel.width - 165; // 165 is the width of the userbar
+		let usedSpace = 0;
+
+		const roomTabs = this.base?.querySelectorAll('ul.siderooms > li');
+		if (!roomTabs) return; // No rooms
+
+		for (const tab of Array.from(roomTabs)) {
+			usedSpace += tab.clientWidth;
```

Comment:
Oh. You're adding them all up individually? Not how I'd do it, but that does work.

Reply:
(I would have just grabbed the rightmost position of the Rooms tab.)